eng/mgmt/Directory.Build.Mgmt.props

```diff
@@ -0,0 +1,3 @@
+﻿<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="CI.Bootstrap.targets" Sdk="Microsoft.Internal.NetSdkBuild.Mgmt.Tools" Version="0.9.0" />
```

Comment:
Should this be split to importing a props and a targets file separately?

Reply:
I deliberately wanted one bootstrap target to do all base importing.
I just didn't want multiple files to do importing rather one file that bootstraps into the nuget package.

Reply:
That might work for now but a lot of times an explicit props and targets import is needed to be able to handle the order in which the properties or targets inserted.

Reply:
Something else that I remembered, as props are processed first prior to processing .targets, I wanted to preempt the importing via props file.
Now you may wonder, then why not simply import CI.Bootstrap.props and have CI.Bootstrap.targets
I am not hard on it being two separate files, all I waned to do is decrease the number of files I have.
If you think for the sake of clarity splitting into two files helps, I am open for doing it and splitting it in two files.

Reply:
Just to be clear props vs targets file extension has not impact on when things are processed, any can be used and only the order of import has impact on when things are processed. However msbuild properties which are global are definitely processed in the order they are seen and targets executed when the target executes. 

The general convention is to have a props and targets file and that would be what I recommend.